presentations/posters/poster_Gagnon_Tahiri/info.json

```
@@ -0,0 +1,15 @@
+{
+    "authors": [
+        {
+            "name": "Justin Gagnon",
+            "affiliation": "Department of Biology, University of Sherbrooke, 2500, boulevard de l'Université, J1K 2R1, QC, Canada"
```

```suggestion
            "affiliation": "Department of Biology, University of Sherbrooke, 2500, boulevard de l'Université, J1K 2R1, QC, Canada",
            "orcid": ""
```
Hi! Could you please use an empty string for author orcid if it cannot be provided?